test/unit/mdc-textfield/mdc-text-field.test.js

```diff
@@ -352,6 +352,47 @@ test('#adapter.deregisterTextFieldInteractionHandler removes an event handler fo
   td.verify(handler(td.matchers.anything()));
 });
 
+test('#adapter.registerValidationAttributeChangeHandler creates a working mutation observer', (done) => {
+  const {root, component} = setupTest();
+  const handler = td.func('ValidationAttributeChangeHandler');
+  td.when(handler(td.matchers.anything(), td.matchers.anything())).thenDo(() => {
+    assert.isNotOk(root.classList.contains('mdc-text-field--invalid'));
```

Comment:
This is testing more logic than I think we want to be testing here; we just want to be testing that the attribute change called this function, right?

I think we might not actually need any assertions, just the `done`. (I'm not sure what kind of timeout is configured right now, if this were to fail...)

Reply:
Timeout is 2000 ms, but the mutationObserver should never take the long. It will just get queued to the next event loop, but I see what you're saying. I could do a `verify(handler(....))`? It seems strange to not assert anything in a test.

Reply:
As discussed, I will remove the assert since testing the removal of .mdc-text-field--invalid is not primary to the test. The fact that the done() is called will cause the test to pass. If it is not called, then the test will result in a failure.